pkgs/tools/typesetting/tectonic/default.nix

```diff
@@ -26,14 +26,15 @@ rustPlatform.buildRustPackage rec {
     owner = "tectonic-typesetting";
     repo = "tectonic";
     rev = "tectonic@${version}";
-    fetchSubmodules = true;
     sha256 = "sha256-xZHYiaQ8ASUwu0ieHIXcjRaH06SQoB6OR1y7Ok+FjAs=";
```

Comment:
Source hash should have been updated here. Fix is https://github.com/NixOS/nixpkgs/pull/377111.